src/eduid_webapp/letter_proofing/views.py

```diff
@@ -119,8 +121,32 @@ def verify_code(user, verification_code):
         current_app.logger.info('Verified code for user {}'.format(user))
         # Remove proofing state
         current_app.proofing_statedb.remove_state(proofing_state)
-        return {'success': True}
+        return {'success': True,
+                'message': 'letter.verification_success',
+                'nins': user.nins.to_list_of_dicts()}
     except AmTaskFailed as e:
         current_app.logger.error('Verifying nin for user {} failed'.format(user))
         current_app.logger.error('{}'.format(e))
         return {'_status': 'error', 'error': 'technical_problems'}
+
+
```

Comment:
We can talk about it tomorrow, I'm not sure. I think we need to rationalize the boundaries of the services.
In any case I added this view in letter proofing b/c that service is already registered to sync the nins attribute to the central db; I don't think we want to register the personal data service to synchronize nins...